datasets/gnad_10/gnad_10.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Ten Thousand German News Articles Dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_DESCRIPTION = """\
+This dataset is intended to advance topic classification for German texts. A classifier that is efffective in
+English may not be effective in German dataset because it has a higher inflection and longer compound words.
+The 10kGNAD dataset contains 10273 German news articles from an Austrian online newspaper categorized into
+9 categories. Article titles and text are concatenated together and authors are removed to avoid a keyword-like
+classification on authors that write frequently about one category. This dataset can be used as a benchmark
+for German topic classification.
+"""
+
+_HOMEPAGE = "https://tblock.github.io/10kGNAD/"
+
+_LICENSE = "Creative Commons Attribution-NonCommercial-ShareAlike 4.0"
+
+_TRAIN_DOWNLOAD_URL = "https://raw.githubusercontent.com/tblock/10kGNAD/master/train.csv"
+_TEST_DOWNLOAD_URL = "https://raw.githubusercontent.com/tblock/10kGNAD/master/test.csv"
+
+
+class GNAD_10(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Gnad10(datasets.GeneratorBasedBuilder):
```
can you rename this class please ?

can you also rename the dataset folder and the dataset script to `gnad10` ?

Reply:
thanks @lhoestq! i've renamed everything to reflect `gnad10`, but i can't figure out how to include changes that are only related to this PR (i'm going to be on the road for a few days but i will be back monday).